README.md

```diff
@@ -15,37 +37,44 @@ or
 flutter pub add web3auth_flutter
 ```
 
-Please refer to the native SDKs for platform-specific configuration.
+## 🌟 Configuration
+
+Add `web3auth_flutter` as a dependency to your `pubspec.yaml`
+
+```yaml
+dependencies:
+  web3auth_flutter:
+    git: https://github.com/torusresearch/web3auth-flutter-sdk.git
+```
+
+#### Android
 
-- [Android SDK](https://github.com/Web3Auth/web3auth-android-sdk)
-- [iOS SDK](https://github.com/Web3Auth/web3auth-swift-sdk)
+- Perform the native [Android integration steps](https://web3auth.io/docs/sdk/android/).
+- For Android, the `redirectUrl` parameter is configurable, and has to be added into the `AndroidManifest.xml`.
 
-For iOS, the redirectUrl parameter is fixed, which is `\(bundleId)://auth`, and does not need to be added as a iOS Custom URL Scheme.
+#### iOS
 
-For Android, the redirectUrl parameter is specifiable, and has to be added into the AndroidManifest.xml.
+- Perform the native [iOS integration steps](https://web3auth.io/docs/sdk/ios/).
 
-## Usage
+- You may add the `redirectUrl`, which is `\(bundleId)://auth`  and does not need to be added as a iOS Custom URL Scheme
 
-Refer to the demo app for more detailed example.
+#### Register the URL scheme you intended to use for redirection
 
-````dart
-// Initialization
- await Web3AuthFlutter.init(Web3AuthOptions(
-        clientId:
-            'BCtbnOamqh0cJFEUYA0NB5YkvBECZ3HLZsKfvSRBvew2EiiKW3UxpyQASSR0artjQkiUOCHeZ_ZeygXpYpxZjOs',
-        network: Network.testnet,
-        redirectUrl: Uri.parse('torusapp://org.torusresearch.flutter.web3authexample'),
-        whiteLabel: WhiteLabelData(
-            dark: true, name: "Web3Auth Flutter App", theme: themeMap)));
-```
+- Android `AndoidManifest.xml` (required)
+- iOS `Info.plist` (optional)
+
+## 🩹 Examples
+
+Checkout the examples for your preferred blockchain and platform in our [examples repository](https://github.com/Web3Auth/examples/)
+
+## 🌐 Demo
 
-// Login
-final Web3AuthResponse response = await Web3AuthFlutter.login(LoginParams(loginProvider: Provider.google));
+Checkout the [Web3Auth Demo](https://demo-app.web3auth.io/) to see how Web3Auth can be used in an application.
```

Comment:
use the demo for flutter

Reply:
there's no hosted demo for flutter

Reply:
I think you can use something similar to Android readme.

> Further checkout the [example folder](https://github.com/Web3Auth/web3auth-flutter-sdk/tree/master/example) within this repository, which contains a sample app.